### PR TITLE
Add instructions to install in venv

### PR DESCRIPTION
**Pull request by luispcunha**
Add an FAQ section to the README, with instructions to install the package in a fresh virtual environment in case the main install fails.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,8 @@ These will provide the default installation of the package, that allow you to su
 pip install --upgrade "inductiva[molecules_extra]"
 ```
 
+If you had trouble installing the package, check the [FAQ](#1-trouble-installing) for more details.
```
Comment (begaiym-k) [inline README.md:33]: have troubles*

File: README.md [hunk 2]
```diff
@@ -170,7 +172,42 @@ All of the simulations will be launched in one go. The user can check the status
...
+```
+C:\> <venv>\Scripts\activate.bat
+```
+
+For `PowerShell` (Windows):
+```
+PS C:\> <venv>\Scripts\Activate.ps1
+```
+
+After activating the virtual environment, you can install the package as described above:
```
Comment (begaiym-k) [inline README.md:209]: above or below?

**Review by begaiym-k — APPROVED** [review]
LGTM! Just left a few minor comments.

File: README.md [hunk 3]
```diff
@@ -170,7 +172,42 @@ All of the simulations will be launched in one go. The user can check the status
...
 
-## FAQ:
+## More info:
+
+- [Managing submitted tasks](https://github.com/inductiva/inductiva/tree/main/inductiva/tasks)
+- [Managing computation resources](https://github.com/inductiva/inductiva/tree/main/inductiva/resources)
+
+## FAQ
+
+### 1. Trouble installing
```
[the remark this answers is not in the document]
Comment (IvanPombo) [reply README.md:182]: Trouble Shooting Installation

**Review by sarmento — APPROVED** [review]
Thank you for this!

**Pull request merged**